frontend/src/pages/login.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html class="no-js" lang="en">
+    
+    <!--#include virtual="includes/head.html" -->
+    
+    <body>
+        <!--#include virtual="includes/header.html" -->
+        <div class="large-12 columns pnl-title pnl-summary">
+            <h1>Log In</h1>
+            <form id="sign-in" method="POST" action="" class="large-4 row">
+                <label for="email">Email:</label>
+                <input type="email" name="email" placeholder="Username" required>
```

Comment:
Can you change the placeholder to `Email Address`? Just to make it explicitly clear it's the email address the user should enter